packages/contracts-bedrock/deploy-config/hardhat.json

```
@@ -18,6 +18,8 @@
   "l2OutputOracleStartingBlockNumber": 1,
   "gasPriceOracleOverhead": 2100,
   "gasPriceOracleScalar": 1000000,
+  "gasPriceOracleBaseFeeScalar": 7600,
+  "gasPriceOracleBlobBaseFeeScalar": 862000,
```

```suggestion
  "gasPriceOracleBaseFeeScalar": 1368,
  "gasPriceOracleBlobBaseFeeScalar": 810949,
```
[mainnet values](https://github.com/ethereum-optimism/superchain-ops/tree/main/tasks/eth/005-2-ecotone-set-gas-config)